tests/test_stump.py

```diff
@@ -146,3 +146,91 @@ def test_two_constant_subsequences_A_B_join():
     right = stump(pd.Series(T_B), m, pd.Series(T_A), ignore_trivial=False)
     utils.replace_inf(right)
     npt.assert_almost_equal(left[:, 0], right[:, 0])  # ignore indices
```

Comment:
We should check if the indices would resolve itself when the input is random. The precision problems really only occur when there are multiple subsequence matches